paddlenlp/transformers/tokenizer_utils_base.py

```diff
@@ -1690,6 +1689,10 @@ def convert_added_tokens(obj):
                 "Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained."
             )
 
+        # save all of related things into default root dir
+        if pretrained_model_name_or_path in cls.pretrained_init_configuration:
+            tokenizer.save_pretrained(default_root)
```

Comment:
Should this only be triggered only when using `init_configuration` and not exist, just like that in `model.from_pretrained`

Reply:
there are many resource files in tokenizer, so I think `checking if the target file exist` will be in `save_pretrained` dir.

Another question, If user modify the tokenizer/model related configuration file, will it run with no issue ?  or , are there some situations that the developer should modify the configuration to suit their model ???

Reply:
Sure, `Configuration` is another issue we need to resolve.